Kibana/variables.tf

```diff
@@ -0,0 +1,32 @@
+variable "test_domain" {
+  default = "cloud-platform-test"
+}
+
+# for clusters, allow all 3 NAT GW IPs
+variable "allowed_test_ips" {
+  type = "map"
+
+  default = {
+    "81.134.202.29/32"  = "office"
```

Comment:
I'm undecided about exposing IP ranges... It doesn't feel like a big risk, so I'm going to go ahead and approve but can you please just check that everyone else concurs @razvan-moj?